otter/test/convergence/test_composition.py

```diff
@@ -25,25 +70,33 @@ def setUp(self):
         Sample server json
         """
         self.servers = [
-            {'id': 'a', 'state': 'ACTIVE', 'created': now(),
-             'addresses': {'private': [{'addr': 'ip1', 'version': 4}]}},
-            {'id': 'b', 'state': 'ACTIVE', 'created': now(),
-             'addresses': {'private': [{'addr': 'ip2', 'version': 4}]}}
+            NovaServer(id='a', state=ServerState.ACTIVE, created=0, servicenet_address='ip1'),
+            NovaServer(id='b', state=ServerState.ACTIVE, created=0, servicenet_address='ip2'),
         ]
 
+    def _get_get_all_convergence_data(self, servers, group_id, reqfunc):
```

Comment:
No specific opinion about choosing the methodology above :) but `_get_gacd_func` name might be more clear?